### ISSUE TITLE
Make it more obvious when you have a Turbo decision to make

### ISSUE THREAD

**Issue by irilyth** [buttonmen-dev/buttonmen#2202]
I'm finding that I keep forgetting to make Turbo decisions before I submit my move; part of it might be that I got used to not having to make them until after hitting BPU, on the old site, but I suspect that it'd help if there was more visually obvious cue when I had a decision to make.

**Comment by irilyth** [comment]
Two suggestions: (1) Turn the background of the selector red rather than white, until you click in the selector; then it's obvious when you have and haven't at least been in the right vicinity. (2) Have the cursor jump to the selector box when you select a Turbo die.

I worry that the second one might be annoying, especially if you were keyboard-navigating through dice. The first sounds to me like it'd be pretty helpful.

We could also put them both as options behind user preferences, if we like adding user preferences.

**Comment by blackshadowshade** [comment]
What about we go back to having the turbo selector invisible when a turbo die hasn't been selected?

**Comment by AdmiralJota** [comment]
A suggestion (4): when you select a turbo die, the selector could flash a for a moment.

**Comment by irilyth** [comment]
It would make it more visibly obvious, for sure. I remember thinking at the time that it was too jarring, so I think I'm fishing for something in between a big change like whole areas appearing and disappearing, and a subtle change from gray to white.

**Comment by blackshadowshade** [comment]
We could always use the blink tag.

/joking

**Comment by blackshadowshade** [comment]
Okay, another suggestion. What about making the background colour of the turbo area white until a turbo die is selected, and then changing the colour to the player colour?

**Comment by irilyth** [comment]
> Okay, another suggestion. What about making the background colour of the turbo area white until a turbo die is selected, and then changing the colour to the player colour?

Ah, do you mean like the whole Turbo area, rather than just the selector? Sure, that might work. I really think it'll be hard to pick a favorite without some mockups -- anyone interested in doing that?

**Comment by blackshadowshade** [comment]
This is actually quite easy to mock yourself, if you're using the Firefox developer tools. Just alter the CSS once the page has loaded.

**Comment by blackshadowshade** [comment]
So, I too am noticing that I'm not noticing the Turbo decision until it's too late. I'll take a look at this issue soon.

**Comment by cgolubi1** [comment]
So... the trivial fix is don't have a default value.  Then the attack fails if the player doesn't add one.

Maybe no one will like this, but it sounds much less complicated than the stuff y'all are talking about, and it kind of makes sense - nothing else has a fixed default during an attack.

**Comment by irilyth** [comment]
One advantage of the default value is to make it easy to say "keep it the same", especially if you don't remember what it was but you know that nothing has changed sine you made the decision to set it before. It'd take up more space to add "(was: N)" text; maybe put in a grayed-out value or something, so you can see it but it's not really in the box? I don't know how to do that, but I feel like I've seen that on other sites (e.g. a box that says "Name" in gray, and when you click there it goes away and you can type your name).

**Comment by cgolubi1** [comment]
Yeah, or show it off to the side as "(was: 7)" or whatever the exact text is we use for swing die setting.

**Comment by irilyth** [comment]
Right, that'd just take up more space. But maybe it'd fit and look nice, dunno.
